packages/block-library/src/term-count/edit.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, BlockControls } from '@wordpress/block-editor';
+import { ToolbarGroup, ToolbarDropdownMenu } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { getBracketIcon } from './icons';
+import { useTermCount } from './use-term-count';
+
+const BRACKET_TYPES = {
+	none: { label: __( 'No Brackets' ) },
+	round: {
+		label: __( 'Round Brackets' ),
+		before: '(',
+		after: ')',
+	},
+	square: {
+		label: __( 'Square Brackets' ),
+		before: '[',
+		after: ']',
+	},
+	curly: {
+		label: __( 'Curly Brackets' ),
+		before: '{',
+		after: '}',
+	},
+	angle: {
+		label: __( 'Angle Brackets' ),
+		before: '<',
+		after: '>',
+	},
```

Comment:
```suggestion
	round: {
		label: __( 'Round brackets' ),
		before: '(',
		after: ')',
	},
	square: {
		label: __( 'Square brackets' ),
		before: '[',
		after: ']',
	},
	curly: {
		label: __( 'Curly brackets' ),
		before: '{',
		after: '}',
	},
	angle: {
		label: __( 'Angle brackets' ),
		before: '<',
		after: '>',
	},
```


In UIs like this, Sentence case is preferred over Title Case. 

<img width="206" height="159" alt="Image" src="https://github.com/user-attachments/assets/ef21817d-6ef0-406e-ab9a-5ef9189d88ef" />

<img width="157" height="116" alt="Image" src="https://github.com/user-attachments/assets/fa732218-5976-4eda-bea7-c057f14895b2" />

See: https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/documentation/copy-guide.md#five-pay-attention-to-capitalization